docs/reference/elasticsearch/configuration-reference/networking-settings.md

```diff
@@ -434,7 +434,9 @@ The compression settings do not configure compression for responses. {{es}} will
 
 ## Advanced remote cluster (API key based model) settings [remote-cluster-network-settings]
```

Comment:
this page has the following front matter:

```
---
mapped_pages:
  - https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html
applies_to:
  deployment:
    ess:
    self:
---
```

I think the applies_to is wrong here (it either needs more or less) 

without cross-referencing with the tutorials I am not sure which one